dev-guide/api-conventions.md

```diff
@@ -0,0 +1,257 @@
+  // +kubebuilder:validation:Required
+  Type string `json:"type"`
+
+  // Documentation allows the user to define documentation for the example.
+  // When this value is provided, the type must be set to either Documentation or Mixed.
+  // The content of the documentation is free form text but must be no longer than 512 characters.
+  // +kubebuilder:validation:MaxLength:=512
+  // +optional
+  Documentation string `json:"documentation,omitempty"`
+
+  // Convention allows the user to define the configuration for this API convention.
+  // For example, it allows them to set the priority over other conventions and whether
+  // this policy should be strictly observed or weakly observed.
+  // When this value is provided, the type must be set to either Convention or Mixed.
+  // +optional
+  Convention ConventionSpec `json:"convention,omitempty"`
+
+  // Author allows the user to denote an author for the example convention.
+  // The author is not required. When omitted, this means the user has no opinion and the value is
+  // left to the platform to choose a good default, which is subject to change over time.
+  // The current platform default is OpenShift Engineering.
+  // The Author field is free form text.
+  // +optional
+  Author string `json:"author,omitempty"`
+}
+```
+
+This API is then explained by `oc explain` as:
+
+```shell
+RESOURCE: example <Object>
+
+DESCRIPTION:
+  Example enables developers to understand how to write user facing documentation within the Godoc of their API types.
+  Example is used within the wider Conventions to improve the end user experience and is a required convention. At
+  least one value must be provided within the example and the type should be set appropriately.
+
+FIELDS:
+  author <string>
+    Author allows the user to denote an author for the example convention. The author is not required. When omitted,
+    this means the user has no opinion and the value is left to the platform to choose a reasonable default, which is
+    subject to change over time. The current platform default is OpenShift Engineering. The Author field is free form
+    text.
+
+  convention <Object>
+    Convention allows the user to define the configuration for this API convention. For example, it allows them to set
+    the priority over other conventions and whether this policy should be strictly observed or weakly observed. When
+    this value is provided, the type must be set to either Convention or Mixed.
+
+  documentation <string>
+    Documentation allows the user to define documentation for the example. When this value is provided, the type must
+    be set to either Documentation or Mixed. The content of the documentation is free form text but must be no
+    longer than 512 characters.
+
+  type <string>
+    Type allows the user to determine how to interpret the example given. It must be set to one of the following
+    values; Documentation, Convention, or Mixed.
```

Comment:
```suggestion
    values: Documentation, Convention, or Mixed.
```

Reply:
Thanks, missed that while updating the other location, have updated now